.env.example

```diff
@@ -1,8 +1,10 @@
-DATABASE_URL=
+PGHOST=
+PGPORT=
+PGUSER=
+PGDATABASE=
+PGPASSWORD=
+COMMENT_SRC_LOC=
```

Comment:
what is this one for?

Reply:
pgocaml, the env vars it uses, this last one is for log somethings that's happening when the queries are called

Reply:
aah got it, would be cool to add as a comment as this is pretty implicit

Reply:
indeed, gonna add